frontend/src/lib/LanguageSelector.svelte

```diff
@@ -0,0 +1,63 @@
+<script>
+  import { Link, Modal } from 'carbon-components-svelte'
```

Comment:
Sort the imports so that first are imports from `carbon-components-svelte`, then `./stores`, then `.json`, and the `.css` imports are the last

And make there be just one `import { ... } from 'carbon-components-svelte'` with all the things you use from there in the one line.